teachopencadd/talktorials/T013_query_pubchem/talktorial.ipynb

```diff
@@ -6,10 +6,13 @@
    "source": [
```

Comment:
<blockquote>In this notebook, you have learned how to access and search similar compounds from the PubChem database via <strong>the</strong> PUG-REST programmatic access.</blockquote><p> </p><blockquote>See <a href="https://pubchemdocs.ncbi.nlm.nih.gov/pug-rest-tutorial" target="_blank">PUG-REST</a> to get more power!  </blockquote><p>Hehe, I like that sentence :D</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>